docs/pages/includes/machine-id/machine-id-init-bot-data.mdx

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
Remove the empty line. If you want to put an empty line before the content, that should be done where the partial is included, not here.